jina/logging/jina_logger.py

```diff
@@ -0,0 +1,252 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
```

Comment:
we don't do recursive module naming, if you don't like `base.py` then rename it to `jina/logging/logger.py`, `jina` should not appear again in the filename